tests/test_from_thread.py

```diff
@@ -628,6 +628,7 @@ async def raise_baseexception() -> None:
             assert exc.value.__context__ is None
 
     @pytest.mark.parametrize("portal_backend_name", get_all_backends())
+    @pytest.mark.usefixtures("deactivate_blockbuster")
     async def test_from_async(
```

Comment:
Blockbuster makes this test fail.
At some point `run_sync_from_thread` is called which calls the blocking  `result()` on a future
I'm not sure exactly what to do.
So deactivating BlockBuster at the moment.
@agronholm Do you have insight ?

Reply:
> At some point run_sync_from_thread is called which calls the blocking result() on a future

This is a context manager that's expected to run from a sync context. The test's conditions shouldn't even occur in normal usage.

Also, the fact that it "blocks" is a feature, because it tests for a regression that might cause a deadlock. See the comment at the test's beginning.

Thus, not a problem. Just deactivate the blockbuster.